include/deal.II/sundials/ida.h

```diff
@@ -581,13 +595,25 @@ namespace SUNDIALS
      * @param mpi_comm MPI communicator
      */
     IDA(const AdditionalData &data     = AdditionalData(),
-        const MPI_Comm &      mpi_comm = MPI_COMM_WORLD);
+        const MPI_Comm        mpi_comm = MPI_COMM_WORLD);
 
     /**
      * Destructor.
      */
     ~IDA();
 
+
+    void
+    set_n_iter(const int n_iter)
+    {
+      this->n_iter = n_iter;
+    }
+
+    int
+    get_n_iter() const
+    {
+      return this->n_iter;
+    }
```

Comment:
The documentation is missing here. Why not make unsigned int?

Reply:
It is needed by sundials as an int.